pinot-controller/src/main/java/org/apache/pinot/controller/helix/core/minion/PinotTaskManager.java

```diff
@@ -139,6 +129,46 @@ public PinotTaskManager(PinotHelixTaskResourceManager helixTaskResourceManager,
     }
   }
 
+  private class ZkTableConfigChangeListener implements IZkChildListener {
+
+    @Override
+    public synchronized void handleChildChange(String path, List<String> tableNamesWithType) {
+      checkTableConfigChanges(tableNamesWithType);
+    }
+  }
+
+  private void checkTableConfigChanges(List<String> allTableNamesWithType) {
+    LOGGER.info("Checking task config changes of tables: {}", allTableNamesWithType);
```

Comment:
This might flood the log (think of a cluster with thousands of tables. Suggest only logging the newly subscribed ones and the ones to be cleaned up